stdlib/socket/src/main/ballerina/src/socket/Module.md

```diff
@@ -1,154 +1,28 @@
-                log:printInfo("Client close: " + caller.remotePort.toString());
-            }
-        } else {
-            log:printError("Unable to read the content", result);
-        }
-    }
-
-    resource function onError(socket:Caller caller, error er) {
-        log:printError("An error occurred", er);
-    }
-}
+socket:Client socketClient = new ({host: "localhost", port: 61598});
 ```
 
-### TCP Client endpoints
-The client endpoint is used to connect to and interact with a socket server. The client can only send the data to the
- server. The client's `callbackService` needs to retrieve the data from the server and do multiple requests/responses between the client and the server.
-
+#### UDP Client endpoints
+The `socket:UdpClient` is used to interact with the remote UDP host and it can be defined as follows:
 ```ballerina
-import ballerina/io;
-import ballerina/socket;
-
-public function main() {
-    socket:Client socketClient = new({ host: "localhost", 
-                                       port: 61598, 
-                                       callbackService: ClientService });
-    string msg = "Hello Ballerina\n";
-    byte[] c1 = msg.toBytes();
-    var writeResult = socketClient->write(c1);
-    if (writeResult is int) {
-        io:println("Number of bytes written: " , writeResult);
-    } else {
-        io:println("Unable to written the content", 
-                        writeResult.detail()["message"]);
-    }
-}
-
-service ClientService = service {
-
-    resource function onConnect(socket:Caller caller) {
-        io:println("connect: ", caller.remotePort);
-    }
-    
-    resource function onReadReady(socket:Caller caller) {
-        var result = caller->read();
-        if (result is [byte[], int]) {
-            var [content, length] = result;
-            if (length > 0) {
-                var str = getString(content);
-                if (str is string) {
-                    io:println(<@untainted> str);
-                } else {
-                    io:println(str.reason());
-                }
-                var closeResult = caller->close();
-                if (closeResult is error) {
-                    io:println(closeResult.detail()["message"]);
-                } else {
-                    io:println("Client connection closed successfully.");
-                }
-            } else {
-                io:println("Client close: ", caller.remotePort);
-            }
-        } else {
-            io:println(result);
-        }
-    }
-    
-    resource function onError(socket:Caller caller, error er) {
-        io:println(er.reason());
-    }
-};
-
-function getString(byte[] content) returns @tainted string | error {
-    io:ReadableByteChannel byteChannel = check io:createReadableChannel(content);
-    io:ReadableCharacterChannel 
-         characterChannel = new io:ReadableCharacterChannel(byteChannel, "UTF-8");
-    return characterChannel.read(50);
-}
+socket:UdpClient socketClient = new;
```

Comment:
We need to add code snippets and describe the usage of this client

Reply:
like https://github.com/daneshk/ballerina/blob/nats-docs-updates/stdlib/messaging/nats/src/main/ballerina/src/nats/Module.md

Reply:
It doesn't need to be complete code. only specify the important codes